src/Pim/index.tsx

```diff
@@ -395,6 +395,25 @@ class Pim extends React.PureComponent {
     const { collectionsAddressBook, collectionsCalendar, collectionsTaskList, addressBookItems, calendarItems, taskListItems } = itemsSelector(this.props);
     const [contacts, events, tasks] = itemValuesSelector(this.props);
 
+    const birthdays = contacts.filter((c) => c.bday).map((c) => {
+      let { year, month, day } = c.bday._time;
+      year = year || '2020';
+      if (month < 10) {
+        month = '0' + month;
+      }
+      if (day < 10) {
+        day = '0' + day;
```

Comment:
Even though the typing of `_time` is any, which is why you can do such things, it's not something we do (changing of type). Additionally, you can simplify all of this to:
```
const bdayTime = c.bday._time;
const year = bdayTime.year ?? '2020';
const month = bdayTime.month.toString().padStart(2, '0');
const day = bdayTime.day.toString().padStart(2, '0');
```